controllers/remote/cluster_cache.go

```diff
@@ -232,7 +236,7 @@ func (t *ClusterCacheTracker) Watch(ctx context.Context, input WatchInput) error
 
 	a, err := t.getClusterAccessorLH(ctx, input.Cluster)
 	if err != nil {
-		return err
+		return ErrCreatingRemoteClusterCache
```

Comment:
We should probably wrap the returned error instead of shadowing it?

Reply:
Wrapped it using ErrCreatingRemoteClusterCache.Error() as removing error constants break apidiff.